parts/linux/cloud-init/artifacts/mariner/mariner-package-update.sh

```diff
@@ -27,6 +27,9 @@ if [ -z "${node_name}" ]; then
     exit 1
 fi
 
+# Azure cloud provider assigns node name as the lowner case of the hostname
```

Comment:
I was actually just grappling with this on some unrelated work that I'm doing, were you able to find any documentation which clearly states this?

Reply:
I couldn't find any documentation. Here is the code I could find:

https://github.com/kubernetes/component-helpers/blob/master/node/util/hostname.go
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/52bfaa6fd236200710cf2fab550a34a802a0fa86/pkg/provider/azure_instances_v1.go#L284